### ISSUE TITLE
dwgplot via GNU Plotutils

### ISSUE THREAD

**Issue by rurban** [LibreDWG/libredwg#229]
Then we can deprecate dwg2SVG and dwg2ps, and rather use plotutils, either via libplot or via plot(1),which supports much more output formats, bitmap and vector. Even real plotting formats, like pcl5 and hpgl/2. But also png, svg, ps, ...

The current idea is to be independent on libplot, and generate ASCII plot(5) format manually,
or if available via the libplot C interface.

**Comment by matndev** [comment]
Any news ? Very interested in converting dwg to jpg

**Comment by rurban** [comment]
No, this would need a lot of work. See the `work/dwgplot` branch

**Comment by paolo-caroni** [comment]
This is a very good idea, have you planned to support PlotStyle (.ctb and .stb)?
I'm writing specification of these type of files that maybe can be interesting, also a simple C PoC for read and edit it.
https://github.com/paolo-caroni/LibrePIA
For the spec see doc folder.
